### PR TITLE
KerioTechnologiesInc.KerioControlVPNClient version 9.4.7290

### PR DESCRIPTION
**Pull request by fortellcz**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/103035)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
InstallationVerification-x64-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230420174406

**Comment by stephengillie** [comment]
Hi @fortellcz,

The installation isn't unattended:
image

Is there an installer switch to bypass this and have it install automatically?

File: manifests/k/KerioTechnologiesInc/KerioControlVPNClient/9.4.7290/KerioTechnologiesInc.KerioControlVPNClient.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: KerioTechnologiesInc.KerioControlVPNClient
+PackageVersion: 9.4.7290
+Installers:
+- InstallerUrl: https://cdn.kerio.com/dwn/kerio-control-vpnclient-win64.exe
+  Architecture: x64
+  InstallerType: exe
+  InstallerSha256: 25F357A4E1A819524890FD2EEC1D4389FAB7562DFB16BFBE438B18CD8CF5E734
+  InstallerSwitches:
+    Silent: /S /v /qn
+    SilentWithProgress: /S /v
```
Comment (stephengillie) [inline manifests/k/KerioTechnologiesInc/KerioControlVPNClient/9.4.7290/KerioTechnologiesInc.KerioControlVPNClient.installer.yaml:13]:
```suggestion
    Silent: /v/qn /l"1033"
    SilentWithProgress: /v/qn /l"1033"
```

**Comment by fortellcz** [comment]
@microsoft-github-policy-service agree

**Comment by fortellcz** [comment]
> @fortellcz please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

File: manifests/k/KerioTechnologiesInc/KerioControlVPNClient/9.4.7290/KerioTechnologiesInc.KerioControlVPNClient.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: KerioTechnologiesInc.KerioControlVPNClient
+PackageVersion: 9.4.7290
+PackageLocale: en-US
+Publisher: Kerio Technologies Inc.
+PackageName: Kerio Control VPN Client
```
Comment (stephengillie) [inline manifests/k/KerioTechnologiesInc/KerioControlVPNClient/9.4.7290/KerioTechnologiesInc.KerioControlVPNClient.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.gfi.com/
PrivacyUrl:   https://www.gfi.com/legal/privacy-policy
PackageName: Kerio Control VPN Client
PackageUrl: https://www.gfi.com/products-and-solutions/network-security-solutions/keriocontrol/resources/vpn#vpn
```

**Pull request closed without merging**